vagrant/postchecks_qat_plugin.sh

```diff
@@ -15,33 +15,36 @@ set -o errexit
 echo "QAT plugin validation"
 
 echo "Ensuring that envoy-qat:devel docker image exists..."
-if [[ -z $(sudo docker images | grep envoy-qat | grep devel) ]]; then
+if ! sudo docker images | grep -E "envoy-qat.*devel"; then
     echo "envoy-qat:devel docker image doesn't exists"
     exit 1
 fi
 
 echo "Verifying intel-qat2-plugin daemonset is available..."
 plugin_daemonset=$( kubectl get daemonset | grep intel-qat2-plugin)
-if [[ $plugin_daemonset ]]; then
-   if [[ $(echo $plugin_daemonset | awk '{print $2}') != $(echo $plugin_daemonset | awk '{print $6}') ]]; then
-      echo "Ther Intel QAT daemonset plugin is not available yet"
-      exit 1
-   fi
+if [[ -n $plugin_daemonset ]]; then
+    if [[ $(echo "$plugin_daemonset" | awk '{print $2}') != $(echo "$plugin_daemonset" | awk '{print $6}') ]]; then
```

Comment:
This needs at least a comment (what are the fields which you are comparing)? Maybe something like this would also work:

    kubectl get pods -l app=intel-qat2-plugin -o json | jq '.items[] | {status: .status.phase}'

Reply:
Agree, I'm going to include something in the next commit.

Reply:
Fixed on 4abb01037aa3d91e093f57f1a25732955d3c91fd